app/src/main/java/com/sun/noteapp/data/source/local/NoteDatabase.kt

```diff
@@ -0,0 +1,85 @@
+package com.sun.noteapp.data.source.local
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import com.sun.noteapp.data.model.Note
+
+class NoteDatabase(private val context: Context)
+    :SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION){
+
+    override fun onCreate(p0: SQLiteDatabase?) {
```

Comment:
```suggestion
    override fun onCreate(db: SQLiteDatabase?) {
```
Tương tự ở dưới, đổi lại tên paramenter cho dễ đọc nhé